features/dd-sdk-android-session-replay/build.gradle.kts

```diff
@@ -52,6 +52,7 @@ android {
 
 dependencies {
     api(project(":dd-sdk-android-core"))
+    api(project(":features:dd-sdk-android-trace"))
```

Comment:
blocking: seems like unnecessary change? I don't see any `com.datadog.android.trace` or `com.datadog.trace` usages in this PR